webpack.config.externals.js

```diff
@@ -23,6 +23,17 @@ const externals = [
     "react-router-dom",
 ];
 
+const externals = {};
+
+for (const name of externalsNames) {
+    // TODO(alex): Pick just one type below, e.g. commonjs2
+    externals[name] = {
+        commonjs: name,
+        commonjs2: name,
+        amd: name,
+    };
+}
```

Comment:
I had started simplifying how we handle externals when publishing in a previous PR and then didn't finish it.  This finishes that work.